Sources/XCHammer/XcodeTargetMap.swift

```diff
@@ -35,7 +35,7 @@ func alwaysIncludePathPredicate(_ path: String) -> Bool {
 /// Return a predicate that determines if a file should be included or not.
 /// We can use this function to filter based on user input.
 func makePathFiltersPredicate(_ paths: Set<String>) -> (String) -> Bool {
-    let recursiveFilters = Set<String>(paths.filter({ $0.hasSuffix("/**") }).map() {
+    let recursiveFilters = Set<String>(paths.filter({ $0.hasSuffix("**") }).map() {
```

Comment:
This seems reasonable - if you want to include all files it will work without specifying paths